From 58c6bce321eb2bbe5df81dbe35709eae6934e4c4 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Fri, 7 Mar 2025 16:31:00 +0000
Subject: [PATCH] ARM: dts: bcm2711: Provide a hook for a WiFi MAC

Add the usual wifi@1 node to the Pi 4 family, so that an alternative
WiFi MAC address can be provided by the wifimac overlay.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/broadcom/bcm2711-rpi-ds.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

--- a/arch/arm/boot/dts/broadcom/bcm2711-rpi-ds.dtsi
+++ b/arch/arm/boot/dts/broadcom/bcm2711-rpi-ds.dtsi
@@ -188,6 +188,13 @@
 
 &mmcnr {
 	interrupts = <GIC_SPI 126 IRQ_TYPE_LEVEL_HIGH>;
+	#address-cells = <1>;
+	#size-cells = <0>;
+
+	brcmf: wifi@1 {
+		reg = <1>;
+		compatible = "brcm,bcm4329-fmac";
+	};
 };
 
 &csi0 {
